src/FluentMigrator.Runner.Core/Generators/Generic/GenericQuoter.cs

```diff
@@ -26,11 +26,11 @@ namespace FluentMigrator.Runner.Generators.Generic
 
     public class GenericQuoter : IQuoter
     {
-        private readonly QuoterOptions _quoterOptions;
+        private readonly QuoterOptions _options;
 
-        public GenericQuoter(QuoterOptions quoterOptions = null)
+        public GenericQuoter([CanBeNull] QuoterOptions options)
```

Comment:
Please see @fubar-coder remarks about having the default constructor still available.  I would not use [CanBeNull] - instead, the default constructor should just initialize the quoter to a default value.  Adding nulls will just complicate future code.